tests/test_tempsensordb.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import os
+from unittest import TestCase, main, skipIf
+from tempDB import TempDB
+
+TEMP_DB = 'temp_tempsensor.db'
+TEMP_TABLE = 'temp_lightclapper'
+PREMADE_DB = 'tests/test_db/test_tempsensor.db'
+PREMADE_TABLE = 'test_tempsensor'
+
+class TestTempDB(TestCase):
+	def setUp(self):
+		self.__db = TempDB(db_file = TEMP_DB, name = TEMP_TABLE)
+		self.__db.manual_enter()
+
+	def tearDown(self):
+		self.__db.manual_exit()
+		if os.path.exists(TEMP_DB):
+			os.remove(TEMP_DB)
+
+	def test_create_table(self):
+		error_msg = 'Table already exists'
+		self.assertFalse(self.__db.table_exists(), error_msg)
+
+		self.__db.create_table()
+		error_msg ='Table does not exist'
+		self.assertTrue(self.__db.table_exists(), error_msg)
+
+	def test_add_good_record(self):
+		record = {'date': '2020-11-22',
+			  'time': '13:51:42',
+		  	  'fanStatus': 1}
+
+		self.__db.create_table()
+		error_msg = 'Record exists unexpectedly'
+		self.assertFalse(self.__db.record_exists(record), error_msg)
+
+		self.__db.add_record(record)
+		error_msg = "Record failed to be added to DB table"
+		self.assertTrue(self.__db.record_exists(record), error_msg)
+
+	def test_add_bad_record(self):
+		record = {'date': '2020-11-22',
+			  'time': '14:11:34',
+			  'fanStatus': 1}
+
+		self.__db.create_table()
+		error_msg = 'Record exists unexpectedly'
+		self.assertFalse(self.__db.record_exists(record), error_msg)
+
+#		self.assertRaises(Exception, self.__db.add_record, record)
```

Comment:
Why is this commented out?

Reply:
While debugging I was running into issues with this line, forgot to uncomment it out. I didn't bother changing anything to the test files as we won't be using those for the final demo. I can go back and fix this issue for the final commit as well.